module-bsp/board/rt1051/bsp/lpm/CpuFreqLPM.cpp

```diff
@@ -17,9 +17,9 @@ namespace bsp
         switch (freq) {
         case CpuClock::CpuClock_Osc_4_Mhz:
             /* Set PERIPH_CLK2_PODF. */
-            CLOCK_SetDiv(kCLOCK_PeriphClk2Div, 2);
+            CLOCK_SetDiv(kCLOCK_PeriphClk2Div, 0);
             /* Set AHB_PODF. */
-            CLOCK_SetDiv(kCLOCK_AhbDiv, 1); // CBCDR
+            CLOCK_SetDiv(kCLOCK_AhbDiv, 6); // CBCDR
```

Comment:
Can you elaborate about this change?

Reply:
Please retest also Bell after this change

Reply:
As I wrote, there are two methods - two prescalers that can be changed to get the frequency at 4 MHz.
I suggest not to change these values - then it will not affect bell:
 
```
case CpuClock :: CpuClock_Osc_4_Mhz:
             / * Set PERIPH_CLK2_PODF. * /
             CLOCK_SetDiv (kCLOCK_PeriphClk2Div, 2);
             / * Set AHB_PODF. * /
             CLOCK_SetDiv (kCLOCK_AhbDiv, 1); // CBCDR 
```